### PR TITLE
css-fonts-Bishan-Webeshete

### PR DESCRIPTION
**Pull request by BishanWebeshete**
Screen Shot 2023-01-25 at 12 02 23 PM

### CODE REVIEW

File: css-fonts/styles.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+body {
+  font-family: 'Oswald', sans-serif;
```
Comment (shawnkost) [inline css-fonts/styles.css:2]: You should be using 2 separate fonts. One for the headings, and one for the rest of the content

**Review by shawnkost — CHANGES_REQUESTED** [review]
Hey Bishan! One edit requested here. You should be using 2 separate fonts

**Comment by BishanWebeshete** [comment]
Screen Shot 2023-01-25 at 12 21 12 PM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**